inst/BS5/assets/pkgdown.scss

```diff
@@ -930,3 +918,41 @@ figure>figcaption.quarto-float-caption-top {
   --mermaid-node-bg-color: RGBA(var(--bs-primary-rgb), 0.1);
   --mermaid-node-fg-color: var(--bs-primary);
 }
+
+// autobrand.css ---------------------------------------------------------
+
+/* Define CSS custom properties for theme colors */
+:root {
+  --autobrand-bg: 255, 255, 255;
+  /* white background */
+  --autobrand-fg: 0, 0, 0;
+  /* black foreground */
+}
+
+/* Bootstrap 5 dark mode support */
+[data-bs-theme="dark"] {
+  --autobrand-bg: 34, 34, 34;
+  --autobrand-fg: 238, 238, 238;
+}
+
+/* SVG filter definitions */
+.autobrand-filters {
+  position: absolute;
+  width: 0;
+  height: 0;
+  overflow: hidden;
+}
+
+/* Target only R-generated plot outputs */
+/* In examples sections (reference pages) */
+.r-plt img,
+/* In vignettes/articles - target images in figure-html directories */
+img[src*="figure-html/"] {
```

Comment:
@hadley do you think `img[src*="figure-html/"]` is going to be a robust way to target figures in vignettes? Could a user update the `figure-html` target directory via a custom pkgdown configuration?

Reply:
I think we'd need to a CSS class to generated plots to be able to target them more specifically.